website/integrations/services/meshcentral/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Integrate with MeshCentral
+sidebar_label: MeshCentral
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is MeshCentral
+
+> MeshCentral is a free, open source, web-based platform for remote device management.
+>
+> -- https://meshcentral.com
+
+## Preparation
+
+The following placeholders are used in this guide:
+
+- `meshcentral.company` is the FQDN of the MeshCentral install.
+- `authentik.company` is the FQDN of the authentik install.
+
+## authentik configuration
+
+Create an OAuth2/OpenID provider with the following parameters:
+
+- Client Type: `Confidential`
+- Redirect URIs: `https://meshcentral.company/auth-oidc-callback`
+- Scopes: OpenID, Email and Profile
+- Signing Key: Select any available key
+
+Note the Client ID and Client Secret values.
+
+Next, create an application, using the provider you've created above.
```

Comment:
This makes me realize something... we have the Wizard now, where you can create an app and then a provider (and now even add bindings!), but is it, in a situation like this, easier/better to first create the provider, and *then* the app (using the old, pre-Wizard way)? I'd like to know your opinion, and I'll check with @BeryJu as well.

Reply:
Thanks for the further info; I wasn't sure if there were certain situations where one really needed to do things the legacy way, and create the provider first. But sounds like the Wizard approach will work no matter what. We hope to make the Wizard the default method soon, so yes I agree would be good to just simplify the steps and link to the Wizard instructions.  And.. then do the same in all the integrations guides, yee-haw.

Reply:
To change it to use the Wizard, let's use [this section](https://docs.goauthentik.io/integrations/services/vmware-vcenter/#authentik-configuration) from the VCenter integration guide as our "template". It provides a section to cover any special notes or info about both the app and the provider. Should do the trick. I guess we should also  add a link to the Wizard docs: https://docs.goauthentik.io/docs/add-secure-apps/applications/manage_apps#instructions (ugh I don't like that title Instructions but oh well).

Reply:
Although... I just saw the section you used for the Komga docs, and I kinda like it too...do as you like for this guide, and then let's collab on creating exactly the block we want for the template.